docs/user/alerting/action-types/index.asciidoc

```diff
@@ -82,3 +82,38 @@ PUT test
     }
 }
 --------------------------------------------------
+
+[float]
+[[preconfigured-connector-alert-history]]
+=== Alert history preconfigured {es} index connector
```

Comment:
This title is a little hard to read.  Is the word "preconfigured" needed?

Alert history {es} index connector